momoka-node/src/data-availability-models/validator-errors.ts

```diff
@@ -147,7 +147,13 @@ export enum MomokaValidatorError {
   POTENTIAL_REORG = 'POTENTIAL_REORG',
 
   /**
-   * unknown error should not happen but catch all
+   * This means there was a new version of the metadata that was not able to be processed.
+   * Update the library to the latest version.
+   */
+  PUBLICATION_NOT_RECOGNIZED = 'PUBLICATION_NOT_RECOGNIZED',
```

Comment:
Added new error type so we return correct error when new version of metadata is released